examples/bri-3/front-end/src/login/Login.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { SRI_BACKEND } from '../constants';
+
+const handleLogin = async () => {
+  if (window.ethereum) {
+    window.ethereum.enable();
+    const mnemonic = await axios.get(`${SRI_BACKEND}/auth/mnemonic`);
+    const accounts = await window.ethereum.send('eth_requestAccounts');
+    const signature = await window.ethereum.request({
+      method: 'personal_sign',
+      params: [mnemonic.data, accounts.result[0]],
+    });
+
+    const input = {
+      message: mnemonic.data,
+      signature,
+      publicKey: accounts.result[0],
+    };
+
+    const tokenRequest = await axios.post(`${SRI_BACKEND}/auth/login`, input);
+
+    if (tokenRequest?.data?.access_token) {
+      console.log(tokenRequest.data.access_token);
+      localStorage.setItem('token', tokenRequest.data.access_token);
```

Comment:
in general its bad practice to store token in local storage but i guess it will do for this

Reply:
Well, there is a mixed approach that people/teams use. Anyways, for the current scope, this can be removed